src/components/ProgressBar/core/progressbar.typings.tsx

```diff
@@ -0,0 +1,3 @@
+export interface IProps {
+  percentage: number
+}
```

Comment:
Follow up on the suggestion:

```suggestion
export interface IProps {
  value: number
  max: number
}
```